src/helpers.js

```diff
@@ -39,7 +39,7 @@ const dedent = function(templateStrings, ...values) {
 	return string
 }
 
-const execCmd = (command, workingDir) => {
+const execCmd = (command, workingDir, trimResult = true) => {
```

Comment:
Related with: 
https://github.com/BetaHuhn/repo-file-sync-action/blob/e1f52b72acabf2f08e9c65a8b753be27cef58f5a/src/git.js#L192-L198

I use this function to read the content of the files to generate the git tree to push to GitHub. I had problems with the execCmd function because of the output.trim(), it breaks the end of the files with this function. For that, I added a third parameter to be able to disable the trim (enabled by default)